garden_ai/base_image_dockerfiles/Dockerfile_jupyter_tf_3.10

```diff
@@ -18,5 +16,6 @@ RUN pip install "tensorflow-io-gcs-filesystem==0.34.0"
 RUN pip install "tensorflow-metadata==1.14.0"
 RUN pip install "tensorflow-probability==0.22.0"
 RUN pip install "tensorstore==0.1.46"
+RUN pip install "tensorflow==2.14.0"
 
-ENTRYPOINT ["jupyter", "notebook", "--notebook-dir=/garden", "--ServerApp.token=791fb91ea2175a1bbf15e1c9606930ebdf6c5fe6a0c3d5bd", "--ip", "0.0.0.0", "--no-browser", "--allow-root"]
+ENTRYPOINT ["jupyter", "notebook", "--notebook-dir=/garden", "--ServerApp.token=791fb91ea2175a1bbf15e1c9606930ebdf6c5fe6a0c3d5bd", "--ip", "0.0.0.0", "--no-browser", "--allow-root"]

```

Comment:
I don't know why these last entrypoint lines show a diff. There's no change in these lines.